### PR TITLE
test: add extension tests to our integration test suite

### PR DESCRIPTION
**Pull request by nicks**
Hello @landism,

Please review the following commits I made in branch nicks/extensiong:

8b52b9164442dcdb8b0f4883a13b34870419f759 (2021-03-08 16:08:54 -0500)
test: add extension tests to our integration test suite

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,6 +77,11 @@ integration-kind:
 	KUBECONFIG="$(kind get kubeconfig-path --name="integration")" go test -mod vendor -p $(GO_PARALLEL_JOBS) -tags 'integration' -timeout 700s ./integration -count 1
 	kind delete cluster --name=integration
 
+# Run the extension integration tests, assuming the extensions
```
Comment (landism) [inline Makefile:80]: I'm not sure how GOPATH is involved. It looks like this is running them from /tmp?

**Review by landism — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (nicks) [reply Makefile]: ooh, good catch, fixed

**Comment by nicks** [comment]
i needed to update this a bit so i could run the dev binary in prod web-mode, PTAL?

File: scripts/test-extensions.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# Checkout extensions to a temporary directory and run tilt against them.
+
+set -ex
+
+DIR=$(mktemp -d)
+git clone git@github.com:tilt-dev/tilt-extensions "$DIR"
```
Comment (milas) [inline scripts/test-extensions.sh:8]: FYI You might want to do `https://github.com/tilt-dev/tilt-extensions` so it'd work without an SSH key, but not a big deal

Comment (nicks) [reply scripts/test-extensions.sh]: good point!

**Pull request merged**